app.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "vali-ah-backend",
+    "description": "1kbIdeas: The one-stop shop for all the self-help ideas you need",
+    "scripts": {},
+    "env": {
+      "NODE_ENV": {
```

Comment:
DATABASE_URL and PORT are also required or don't they need to be included?

Reply:
I really do not know. Can you please provide more insight on that?